model/godleyIcon.h

```diff
@@ -33,15 +33,70 @@
 namespace minsky
 {
   class GodleyTableEditor;
+  
+  // Godley variables broken out in a separate structure, as copying is non-default
+  class GodleyVars: public ItemT<GodleyVars>
+  {
+  public:	    
+    typedef std::vector<VariablePtr> Variables;
+    const Variables& flowVars() const {return m_flowVars;}
+    const Variables& stockVars() const {return m_stockVars;}    	  
+	  
+    GodleyVars() {}
+    virtual ~GodleyVars() {}
+    // copy operations not deleted to allow ItemT<Group> to compile
+    GodleyVars(const GodleyVars& x) {};
+    GodleyVars& operator=(const GodleyVars&) {return *this;}
```

Comment:
If the only members are the two variable vectors, then the copy operations should be just a simple call to update()